.github/workflows/emojify-pr.yml

```diff
@@ -1,6 +1,13 @@
 name: Emojify PR Title
 
-on: pull_request
+# delete lines 5-8 and uncomment line 11 to to enable the Emojify PR Title workflow
+
+on:
+  pull_request:
+    branches-ignore:
+      - "**"
+
+# on: pull_request
```

Comment:
Is it line 10 or line 11? on github preview, it shows line 10

Reply:
ope yes ty